index.js

```diff
@@ -11,6 +11,21 @@ exports.start = function (controller, config) {
   // expose botkit workers
   beepboop.workers = beepBoopBotkit.workers
 
+  // Returns a botkit bot by Slack team id
+  beepboop.botByTeamId = function (teamId) {
+    // Get a handle on the team's botkit worker/bot
+    var resourceId = Object.keys(beepboop.workers).filter(function (key) {
+      var entry = beepboop.workers[key]
```

Comment:
Maybe make this line `var entry = beepboop.workers[key] || {}` in case the entry for `beepboop.workers[key]` is undefined